frontend/src/libraries/explorer-wamp/transactions.ts

```diff
@@ -243,9 +249,13 @@ export default class TransactionsApi extends ExplorerApi {
         };
       } else {
         const transactionExtraInfo = await this.call<any>("nearcore-tx", [
-          transactionHash,
+          transactionInfo.hash,
           transactionInfo.signerId,
         ]);
+        transactionInfo.status = Object.keys(
+          transactionExtraInfo.status
+        )[0] as ExecutionStatus;
+
```

Comment:
We also need the `actions` extracted from the `transactionExtraInfo`, as Explorer Backend strips DeployContract contract binary and FunctionCall args from the data and don't save it to the DB.

We used to do this on getTransactions:
https://github.com/near/near-explorer/blob/74b1cb84f4821b54a2b8e5fe20c06e1a16f15c10/frontend/src/libraries/explorer-wamp/transactions.ts#L192-L209

We don't need to do it on getTransactions since actions saved on the backend to the DB includes enough information for the pages where we display lists of transactions, but on the Transaction Details page we need more, hence the need to fetch an parse the data from RPC.